app/helpers/application_helper.rb

```diff
@@ -2,4 +2,7 @@
 
 # アプリケーション共通のヘルパ
 module ApplicationHelper
+  def exit_admin_manager
+    Manager.exists?(role: 'admin')
+  end
```

Comment:
Admin  権限のマネージャーが存在した場合登録画面を表示させない用で使っています。

Reply:
rubyの書き方として、メソッドの返り値がboolean型の場合はメソッド名?という形にするのが通例です。そうするとこのメソッドはtrueかfalseを返すんだなと他のエンジニアが理解しやすくなります。メソッドの名前自体も、間違ってますかね？`admin_manager_exists?`が良さそうです。

```suggestion
  def admin_manager_exists?
    Manager.exists?(role: 'admin')
  end
```

Reply:
@shotaimai66 
こちらメソッド名も完全に間違いでした！
> メソッドの返り値がboolean型の場合はメソッド名?という形にするのが通例です。
これはとても勉強になりました！ありがとうございます！